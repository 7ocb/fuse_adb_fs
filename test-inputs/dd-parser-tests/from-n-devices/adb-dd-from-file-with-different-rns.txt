
akdlfj
lkdfjlasdj
asdlfkj
there no \r
adsf
as
again no \r
df
there only \rds and again only \rf
sadf
ds
0+1 records in
0+1 records out
117 bytes transferred in 0.001 secs (117000 bytes/sec)
