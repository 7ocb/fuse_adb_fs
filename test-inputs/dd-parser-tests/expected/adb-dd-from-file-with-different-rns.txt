
akdlfj
lkdfjlasdj
asdlfkj
there no \r
adsf
as
again no \r
df
there only \rds and again only \rf
sadf
ds
